kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -1737,6 +1737,16 @@ object KyuubiConf {
       .stringConf
       .createOptional
 
+  val SERVER_INFO_PROVIDER: ConfigEntry[String] =
+    buildConf("kyuubi.server.info.provider")
+      .doc("The server information provider name, some clients may rely on this information" +
+        " to check the server compatibilities and functionalities." +
+        " <li>SERVER: Return Kyuubi server information.</li>" +
+        " <li>ENGINE: Return Kyuubi engine information.</li>")
+      .version("1.6.1")
+      .stringConf
+      .createWithDefault("SERVER")
```

Comment:
If the target branch is master, the default value should be `ENGINE`.
If the target branch is branch-1.6, the default value should be `SERVER`.

Reply:
I'm planing to merge this one to master/1.6 first, and open a new PR for master to change the default value to ENGINE.